### ISSUE TITLE
TOP PRIORITY: Check image for a11y before posting

### ISSUE THREAD

**Issue by VickiLanger** [VickiLanger/LGBTQ-of-the-day-bot#250]
**Describe the bug**
Sometimes text just isn't legible and therefore not accessible. 

**Expected behavior**
Background and text should be verified to be WCAG compliant together before being allowed to post

**Screenshots**

![image with text that is def not accessible with the background](https://user-images.githubusercontent.com/46823403/97648592-8e679100-1a2b-11eb-8125-34785e39d8f4.jpeg) 

https://twitter.com/lgbtqotd/status/1321920117551353856

**Additional context**

A suggested fix I was given:
- [read rgb value of pixel wit python pillow](https://stackoverflow.com/a/27960627)
- then use the map function to apply the side by side contrast formula
- Then use map again to set the pixels meeting the contrast threshold to white and the rest to black, then flatten the numpy array and feed back to the image library to make a contrast image.

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/LGBTQotd) to see the hard work you put in.*

**Comment by LuNoX** [comment]
@VickiLanger do you still need help with this? :)

**Comment by VickiLanger** [comment]
> @VickiLanger do you still need help with this? :)

This definitely still needs work. You're welcome to it if you like

**Comment by laksh-krishna-sharma** [comment]
Hello @VickiLanger may i work on this issue with a bit of your guidance
